docs/errors.md

```diff
@@ -0,0 +1,41 @@
+# Common Errors
+
+This page should be used as a summary of common errors encountered while
+developing the project. Have a look at it, when you're encountering some weird
+problem, maybe it's listed here and if not, consider adding it.
+
+## Docker Compose Error
+
+The `docker-compose.dev.yml` located in the `.devcontainer` uses **version
+3.8**. The version describes the yml notation, the supported syntax and
+commands.
```

Comment:
The `version` element [has been deprecated and is now purely informative for backwards compatibility](https://docs.docker.com/compose/compose-file/#version-top-level-element).